pkgs/development/python-modules/hassil/default.nix

```
@@ -37,10 +34,12 @@ buildPythonPackage {
     unicode-rbnf
   ];
 
+  pythonImportsCheck = [ "hassil" ];
+
   nativeCheckInputs = [ pytestCheckHook ];
 
   meta = with lib; {
-    changelog = "https://github.com/home-assistant/hassil/blob/${version}/CHANGELOG.md";
+    changelog = "https://github.com/home-assistant/hassil/blob/${src.tag}/CHANGELOG.md";
```

Probably simpler to just say

```suggestion
    changelog = "https://github.com/home-assistant/hassil/blob/v${version}/CHANGELOG.md";
```

---

Disagree, the `src.tag` is exactly what we want at that place in the URL.

---

Okay then.